lib/assert.js

```diff
@@ -546,4 +546,8 @@ function doesNotThrow(block, /*optional*/error, /*optional*/message) {
   _throws(false, block, error, message);
 }
 
-assert.ifError = function ifError(err) { if (err) throw err; };
+assert.ifError = function ifError(err, message) {
+  if (!err) return;
+  message = message || `ifError failed. Original error:\n${err}`;
```

Comment:
Hm, is `ifError` really failing here? Maybe smth. like `ifError encountered unexpected error: ...` might be more intuitive?